File: customview/src/main/java/dinson/customview/activity/SplashActivity.java
package dinson.customview.activity;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import dinson.customview.R;

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        startActivity(new Intent(this, Main2Activity.class));
        finish();
    }
}

File: customview/src/main/java/dinson/customview/activity/SplashActivity.kt
package dinson.customview.activity

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import dinson.customview.R

class SplashActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        startActivity(Intent(this, MainActivity::class.java))
        finish()
    }
}
